frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -125,8 +126,8 @@ const SampleDetailsContent = ({
   let processMeasurements = []
   let experimentRunsIDs = []
   const library = librariesByID[id]
-  const quantity = library && library.quantity_ng ? parseFloat(library.quantity_ng).toFixed(3) : undefined
-  const concentration_nm = library && library.concentration_nm ? parseFloat(library.concentration_nm).toFixed(3) : undefined
+  const quantity = library && library.quantity_ng ? NumberFormat(parseFloat(library.quantity_ng), 1) : undefined
+  const concentration_nm = library && library.concentration_nm ? NumberFormat(parseFloat(library.concentration_nm), 1) : undefined
```

Comment:
I am not sure all numeric values should share the same precision (this would need to be checked with the lab. Also if we apply a precision in one list it should be done everywhere. The library list and project sample list should also share the same changes.